tests/sentry/hybrid_cloud/test_rpc.py

```diff
@@ -0,0 +1,48 @@
+from django.test import override_settings
+
+from sentry.services.hybrid_cloud.organization import (
+    OrganizationService,
+    RpcOrganizationMemberFlags,
+)
+from sentry.services.hybrid_cloud.organization.impl import DatabaseBackedOrganizationService
+from sentry.services.hybrid_cloud.rpc import dispatch_to_local_service
+from sentry.services.hybrid_cloud.user import RpcUser
+from sentry.silo import SiloMode
+from sentry.testutils import TestCase
+
+
+class RpcServiceTest(TestCase):
+    def test_remote_service(self):
```

Comment:
would also like to see coverage for bad invocations. What happens when you call a non @rpc_method in each region?

Reply:
Can you elaborate on the kind of bad invocations? Methods not tagged with `@rpc_method` just work normally, and are mostly stuff like serialization helpers.

If you're looking at the `add_organization_member` call below, that was an arbitrary example I picked to exercise the scaffolding. I could add a region-only one to go with it.

If you mean bad invocations like passing serial arguments with bogus parameter names or the wrong types, we won't really be set up to handle those until we've squashed the remaining bugs that trip the error condition [here](https://github.com/getsentry/sentry/blob/62c477285b18b4c59a54ea36488a420e9f7dea87/src/sentry/services/hybrid_cloud/rpc.py#L241-L246). I intend to tackle that when wiring it to an API endpoint, because that's where we'd be receiving the bad invocation and passing back an error response.